### PR TITLE
feat: add validation to qualifying questions

### PR DESCRIPTION
**Pull request by SDSheriffShawnVahle**
# Description

Add validation to qualifying questions tab. 

Fixes # issue ticket number and link

## Type of change
[AB#2605](https://dev.azure.com/dsd-sdsd/c59f7cfe-fbe6-43ba-85f4-23bca3c651c6/_workitems/edit/2605)

### CODE REVIEW

File: UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <v-card-text class="text-right">
       <v-btn
         outlined
-        :color="$vuetify.theme.dark ? 'accent' : 'accent'"
+        :color="$vuetify.theme.dark ? 'primary' : 'primary'"
```
Comment (jacobkellas) [inline UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue:35]: you can just make this color="primary"

**Review by jacobkellas — CHANGES_REQUESTED** [review]

File: UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue (hunk 1 above)
Comment (SDSheriffShawnVahle) [reply UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue]: Done.

File: UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue [hunk 2]
```diff
@@ -32,7 +32,7 @@
     <v-card-text class="text-right">
       <v-btn
         outlined
-        :color="$vuetify.theme.dark ? 'accent' : 'accent'"
+        :color="'primary'"
```
Comment (jacobkellas) [inline UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue:35]: take out the binding, just 'color="primary"'

**Review by jacobkellas — CHANGES_REQUESTED** [review]

File: UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue (hunk 2 above)
Comment (SDSheriffShawnVahle) [reply UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue]: Done.

File: UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue [hunk 3]
```diff
@@ -32,7 +32,7 @@
     <v-card-text class="text-right">
       <v-btn
         outlined
-        :color="$vuetify.theme.dark ? 'accent' : 'accent'"
+        color="'primary'"
```
Comment (jacobkellas) [inline UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue:35]: now take out the inner quotes :)

**Review by jacobkellas — CHANGES_REQUESTED** [review]

File: UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue (hunk 3 above)
Comment (SDSheriffShawnVahle) [reply UI/libs/core-public/src/lib/components/wrappers/GeneralInfoWrapper.vue]: One of these days i'll get this right lol. Done.

**Review by jacobkellas — APPROVED** [review]

**Pull request merged**